release/files_common

```diff
@@ -229,6 +229,8 @@ selfdrive/locationd/generated/ubx.cpp
 selfdrive/locationd/generated/ubx.h
 selfdrive/locationd/generated/gps.cpp
 selfdrive/locationd/generated/gps.h
+selfdrive/locationd/generated/glonass.cpp
+selfdrive/locationd/generated/glonass.h
```

Comment:
you can replace all the generated/ lines here with
```suggestion
selfdrive/locationd/generated/*.cpp
selfdrive/locationd/generated/*.h
```

Reply:
added in https://github.com/commaai/openpilot/pull/27125
which needs to be merged first